Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Editor.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: 353ad08d551b37148bdb67acc7676edc
+folderAsset: yes
+DefaultImporter:
+  externalObjects: {}
```

Comment:
Documentation update?

Reply:
Added some info to the WMR camera settings docs